Old Generated/Parquet Designer.csv

```diff
@@ -0,0 +1,26 @@
+ID,Name,Subtype,TrenchAdjective,AddsToDesert,AddsToForest,AddsToHeavens,AddsToVolcanic,AddsToSeaside,AddsToSwamp,AddsToTundra,IsWalkable,IsHole,IsFlammable,IsLiquid,ModTool,GatherTool,MaxToughness

```

Comment:
Do you want to keep the Old Generated folder around?

Reply:
Yes, for now.  It'll be deleted once we know that CSV serialization is working.